docs/components/drawer/index.html

```diff
@@ -124,4 +125,128 @@ <h3 v-text="title"></h3>
         </footer>
       </div>
     </section>
+
+    <section>
+        <header>
+            <h2 id="drawer-positionable-elements">Drawer with Positionable Child Elements</h2>
+            <p>
+              <hx-icon type="info-circle"></hx-icon>
+               Positionable elements (e.g., <code>&lt; hx-tooltip &gt;</code> and
+               <code>&lt; hx-popover &gt;</code>) need to be direct children of the
+               <code>&lt; hx-drawer &gt;</code> element, and placed in a named slot.
```

Comment:
I made a mistake for this paragraph.  Please remove the extra spaces around the `<code>` blocks.

```html
<code>&lt;hx-tooltip&gt;</code>
<code>&lt;hx-popover&gt;</code>
<code>&lt;hx-drawer&gt;</code>
```

So, when rendered look like custom element tags:
`<hx-tooltip>`
`<hx-popover>`
`<hx-drawer>`

Reply:
Changes have been done, please review.